source/basic.tex

```diff
@@ -824,7 +824,7 @@
 \grammarterm{elaborated-type-specifier}{s}\iref{dcl.type.elab} and
 friend declarations\iref{class.friend} may introduce a (possibly not
 visible) name into an enclosing namespace; these restrictions apply to
-that region. Local extern declarations\iref{basic.link} may introduce
+that region. Local \tcode{extern} declarations\iref{basic.link} may introduce
```

Comment:
The intent is to also cover block-scope function declarations, and "local" by definition excludes non-automatic storage duration entities, so this is still wrong :)

How about "Function and \tcode{extern} variable declarations at block scope\iref{basic.link} may introduce [...]".

Reply:
Well... local variable doesn't :P, but that still doesn't cover functions. Applied!